### ISSUE TITLE
Couldn't able to run LinkNeighborloader due to the backend issue, please explain its urgent

### ISSUE THREAD

**Issue by ishanki19-pixel** [pyg-team/pyg-lib#325]
### 🐛 Describe the bug

`
![Screenshot (625)](https://github.com/pyg-team/pyg-lib/assets/81527206/ae566fb6-9280-4e5d-8b2d-23c3b8b3b727)

edge_label_index = train_data["user", "rates", "movie"].edge_label_index
edge_label = train_data["user", "rates", "movie"].edge_label

train_loader = LinkNeighborLoader(
    data=train_data,
    num_neighbors=[20, 10],
    neg_sampling_ratio=2.0,
    edge_label_index=(("user", "rates", "movie"), edge_label_index),
    edge_label=edge_label,
    batch_size=128,
    shuffle=True,
)

# Inspect a sample:
sampled_data = next(iter(train_loader))

print("Sampled mini-batch:")
print("===================")
print(sampled_data)

assert sampled_data["user", "rates", "movie"].edge_label_index.size(1) == 3 * 128
assert sampled_data["user", "rates", "movie"].edge_label.min() == 0
assert sampled_data["user", "rates", "movie"].edge_label.max() == 1`

### Environment

* `pyg-lib` version:
* PyTorch version:
* OS:
* Python version:
* CUDA/cuDNN version:
* How you installed PyTorch and `pyg-lib` (`conda`, `pip`, source):
* Any other relevant information:

**Comment by ishanki19-pixel** [comment]
@rusty1s please resolve my issue as soon as possible, i have my presentation this monday!

**Comment by rusty1s** [comment]
How did you install `pyg-lib` and which version?

**Comment by ishanki19-pixel** [comment]
I have installed using this command:
!pip install --upgrade pyg-lib -f https://data.pyg.org/whl/nightly/torch-2.2.0+cpu.html

**Comment by ishanki19-pixel** [comment]
actually since 2months eveyrthing was working fine
just today itself I encountered this error

**Comment by ishanki19-pixel** [comment]
I was using the same code ...and everything was working fine but idk how it happened

**Comment by ishanki19-pixel** [comment]
@rusty1s  I was using the same installation since so many days without facing any errors!!
Using this below command:
!pip install --upgrade pyg-lib -f https://data.pyg.org/whl/nightly/torch-2.2.0+cpu.html

**Comment by rusty1s** [comment]
What's the output of
```python
import pyg_lib
import torch

inputs = torch.randperm(100_000)
pyg_lib.ops.index_sort(inputs)
```
Does that run through?

**Comment by ishanki19-pixel** [comment]
![image](https://github.com/pyg-team/pyg-lib/assets/81527206/e0b5209d-9f37-44a9-8564-977cf249b515)
showing the same error with your code snippet too!!!

**Comment by ishanki19-pixel** [comment]
@rusty1s please see the above!!!

**Comment by ishanki19-pixel** [comment]
![image](https://github.com/pyg-team/pyg-lib/assets/81527206/919fef3f-6b50-4b9c-8870-d89102845712)
Still showing the same error!! @rusty1s

**Comment by rusty1s** [comment]
Yes, but the error will disappears inside `LinkNeighborLoader` (the original issue).

**Comment by ishanki19-pixel** [comment]
@rusty1s please see the above

**Comment by rusty1s** [comment]
Can you post a reproducible example here (no screenshot)? It's hard to tell what might have gone wrong.

**Comment by rusty1s** [comment]
You can also share the colab with me if that is easier.

**Comment by ishanki19-pixel** [comment]
https://colab.research.google.com/drive/1uR19Ym_q4SUpSCR623IF345zTngyaXqW?usp=sharing

I am sharing the link for colab notebook

**Comment by rusty1s** [comment]
The issue is that you are re-installing torch. Colab comes with PyTorch pre-installed, so simply do
```
!pip install torch_geometric torch_scatter torch_sparse torch_cluster torch_spline_conv -f https://data.pyg.org/whl/torch-2.3.0+cu121.html
!pip install --upgrade pyg-lib -f https://data.pyg.org/whl/nightly/torch-2.3.0+cu121.html
```
```
import pyg_lib
inputs = torch.randperm(100_000)
pyg_lib.ops.index_sort(inputs)
```
then works

**Comment by ishanki19-pixel** [comment]
okayy thank you so much!!! its working now
